README.md

```diff
@@ -200,6 +200,51 @@ import { renderRichText } from "@storyblok/js";
 const renderedRichText = renderRichText(blok.richtext);
 ```
 
+You can set a **custom Schema and component resolver globally** at init time by using the `richText` init option:
+
+```js
+import { richTextSchema, storyblokInit } from "@storyblok/js";
+
+const mySchema = deepClone(richTextSchema) // you can make a copy of the default richTextSchema
```

Comment:
Maybe we could do `Object.assign({}, richTextSchema)` so it's more a copy/paste example?